### ISSUE TITLE
to run the program by sudo 

### ISSUE THREAD

**Issue by icltc** [CustomComputingGroup/MLO#18]
when I ran the program in the first time, it reported this:  
File "/home/kyah/MLO-master/model/trials/trial.py", line 91, in initialise
    results_folder, images_folder = self.create_results_folder()
## ValueError: too many values to unpack

when I found the code in the find and add a return value to this function, it turned out a new error message
File "/home/kyah/MLO-master/model/run.py", line 110, in save_run_data
    with open(self.results_folder_path + '/run_data.txt', 'w') as run_data:
## IOError: [Errno 2] No such file or directory: '/data/log/2013-01-31_22-38-41/run_data.txt'

then I try sudo sh run_mlo,sh, it works....
I believe it's because it cannot create a folder without a sudo

**Comment by icltc** [comment]
or probably we can change the place where store the result file...

**Comment by mk306** [comment]
yes you can! and you should :) in the configuration file change results_folder_path this bit to whatever you want - dont just sudo!!!
